lib/protocol/sequences/Sequence.js

```diff
@@ -43,7 +43,7 @@ Sequence.prototype.hasErrorHandler = function() {
 };
 
 Sequence.prototype._packetToError = function(packet) {
-  var code = ErrorConstants[packet.errno] || 'UNKNOWN_CODE_PLEASE_REPORT';
+  var code = ErrorConstants[packet.errno] || 'UNKNOWN_CODE_' + packet.errno + '_PLEASE_REPORT';
```

Comment:
The other alternative i've considered was:

```suggestion
  var code = ErrorConstants[packet.errno] || 'UNKNOWN_CODE_PLEASE_REPORT: ' + packet.errno;
```

Reply:
I think your alternative is better because any error handling code just has to check for the static string "UNKNOWN_CODE_PLEASE_REPORT".

Reply:
@vlasky applied.

Do you have the power to merge and release this?